CHANGES.rst

```diff
@@ -5,6 +5,12 @@ CHANGES for Crate.io Documentation Theme
 Unreleased
 ----------
 
+2020/01/09 0.7.3
```

Comment:
can you remove this subtitle? the release number and release date are added during the release. changelog entries should normally just be put under the unreleased subtitle (above here)